### ISSUE TITLE
High CPU usage running quasar dev

### ISSUE THREAD

**Issue by MikeyAlder** [quasarframework/quasar#6510]
**Describe the bug**
When running "quasar dev" the CPU in my MacBook Pro usage jumps to over 200-300% in activity monitor for node and the laptop fans go into overdrive.

**Expected behavior**
Should the CPU usage drop once the app has been initially built and is now just serving?

**Platform (please complete the following information):**
OS: Mac OS 10.15.3 (Catalina)
Node: 13.8.0
NPM: 6.13.7
Yarn: 1.22.0
Browsers: Chrome

**Comment by MikeyAlder** [comment]
I just created a new quasar project and it doesn't do it, so it must be something in my project that is causing the issue. Apologies.

**Comment by hawkeye64** [comment]
@MikeyAlder Have you made any changes that affects the linter? I have seen some issues lately with run-away eslint, causing 100% CPU.

**Comment by MikeyAlder** [comment]
@hawkeye64 After checking each package I've loaded individually, I've narrowed it down to the [@fortawesome/vue-fontawesome](https://github.com/FortAwesome/vue-fontawesome) package. They have a [closed ticket on their GitHub repo](https://github.com/FortAwesome/vue-fontawesome/issues/69) with other users that have the same issue.

Switching back to the CSS version of font-awesome fixed the issue, but obviously doesn't allow the tree shaking to reduce file sizes. Hopefully, they will fix the issue soon.
